static/app/views/performance/transactionSummary/transactionReplays/useReplaysFromTransaction.tsx

```diff
@@ -12,6 +12,11 @@ import fetchReplayList, {
   REPLAY_LIST_FIELDS,
 } from 'sentry/utils/replays/fetchReplayList';
 import useApi from 'sentry/utils/useApi';
+import {ReplayListRecord} from 'sentry/views/replays/types';
+
+export type ReplayItem = ReplayListRecord & {
+  txEvent?: {[x: string]: any};
+};
```

Comment:
i don't think `txEvent` should be optional on this type, it looks like it's actually `txEvent: undefined | ...`. Within this file the objects will for sure have `txEvent`. There's a related note in replayTable.tsx

It's so verbose, but `ReplayListItemWithTxn` might be a more descriptive name, especially if we endup with other relations added in the future, like `ReplayListWithError` for example.